src/util/websocket/browser_websocket.js

```diff
@@ -4,7 +4,11 @@ var WS = global.MozWebSocket || global.WebSocket;
 
 module.exports = {
   create: function(url, protocols, options) {
-    if (typeof WS !== 'function') return null;
+    if (typeof WS !== 'function'
+        // In Safari it is an 'object'
```

Comment:
Ran into this bug also. It's an object in Safari 9 (at least in 9.1.1), but in Safari 10 it's fixed and is 'function' as it should be.